kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -206,28 +212,47 @@ define([
                 });
         }
 
+        function addRow(){
+            $(`.${cssBaseClass}__table`).append(
+                tr({
+                    class: `${cssBaseClass}__table_row`,
+                    dataElement: `${cssClassType}-fields-row`,
+                })
+            );
+
+            let parameterRows = ui.getElements(`${cssClassType}-fields-row`);
+
+            parameterRows.forEach((parameterRow, index) => {
```

Comment:
Need to re-write this a bit to only render/add the new row instead or re-rendering all rows.

Reply:
seems like you could just append the new row as long as we know the correct # to display (i think?)

Reply:
Fixed :)